ui/public/i18n/en/translation.json

```diff
@@ -75,7 +75,12 @@
         "pecbridge_admin_mail_format": "Must be a valid email address",
         "phonebook_instances": "Synchronize Phonebook with NethVoice instance",
         "choose_phonebook_instance": "Choose a NethVoice instance in the list",
-        "no_phonebook_instance": "Disable Phonebook synchronization"
+        "no_phonebook_instance": "Disable Phonebook synchronization",
+        "lets_encrypt_tips": "This option enables automatic request of a Let's Encrypt TLS certificate for this app. Manage the certificate from the TLS Certificates page.",
+        "cannot_obtain_certificate": "Cannot obtain a Let's Encrypt certificate for {hostname}",
+        "go_to_tls_certificates": "Go to TLS Certificates",
+        "lets_encrypt_disabled_warning_description": "The certificate might be removed. The app may temporarily become unreachable until a valid certificate is restored",
```

Comment:
Maybe it's better to use a message similar or equal to the one that appears whene deleting a route or a certificate